### ISSUE TITLE
Command \\fileinfo appends an extra space

### ISSUE THREAD

**Issue by thom-sch** [doxygen/doxygen#9550]
**Describe the bug**
The new command `\\fileinfo` (s.a. issue #7946) appends an extra space which prevents f.e. altering the extension to refer to other file types; for an use case see attached zipfile.
 
**Desired behavior**
No extra space appended. 

**To Reproduce**
See attached zip-file:
[Test19A_fileinfo.zip](https://github.com/doxygen/doxygen/files/9444698/Test19A_fileinfo.zip)

Module1.md shall contain further documentation for Module1 and thus shall include code-snippets from Module1.pas and Module1.hpp. Using `\\fileinfo` fails due to the extra space between the output of `\\fileinfo` and the extensions .pas resp .hpp;
Additonally this causes Doxygen to report a warning "included file Modul1 is not found" on trying to include the .pas- and the .hpp module, due to this extra space the desired extension was not closely appended.

**Version**
- 1.9.5 (2f6875a5ca481a69a6f32650c77a667f87d25e88)
- Windows 10

**Additional Info**
Command `\\lineinfo` also appends an extra space (which doesn't matter; I think in most cases its just a flaw).

Regards from Meerbusch
Thomas

[Test19A_fileinfo.zip](https://github.com/doxygen/doxygen/files/9444699/Test19A_fileinfo.zip)

**Comment by albert-github** [comment]
I checked with the doxygen version 1.10.0 and got the warnings:
```
.../Modul1.md:13: warning: Unknown option specified with \fileinfo: 'file'
.../Modul1.md:14: warning: Unknown option specified with \fileinfo: 'xxxx'
.../Modul1.md:54: warning: included file 'Modul1' is not found. Check your EXAMPLE_PATH
.../Modul1.md:60: warning: included file 'Modul1' is not found. Check your EXAMPLE_PATH
.../Modul1.md:54: warning: included file 'Modul1' is not found. Check your EXAMPLE_PATH
.../Modul1.md:60: warning: included file 'Modul1' is not found. Check your EXAMPLE_PATH
```
The first 2 are correct, the other 4 are the ones as described in the issue.

When running with the current master () only the first 2 warnings remain and the output looks OK.
The fact that the `Modul1.pas` (from `\fileinfo{name}.pas`is displayed an empty box is due to the fact that the line
```
\until END; // TClass1
```
should read
```
\until END;
```

in that case all is working as expected.

Looks like to be fixed with (pull request #10580):
```
Commit: 1d6bf94be272230125d930a6ffd8f731ab1ba71c [1d6bf94]
Date: Saturday, January 20, 2024 1:39:09 PM

No extra space after `fileinfo` and `lineinfo` commands

Don't invent a space after the `fileinfo` and `lineinfo` command as the space prohibits concatenations
```

I think the problem is fixed but not released.
(please don't close the issue as this will be done at the moment of an official release).

**Comment by thom-sch** [comment]
Just checked - and works fine :)
Thanks a lot.

Regards from Meerbusch
Thomas

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.11.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
